### PR TITLE
Improve html report

### PR DESCRIPTION
**Pull request by fviernau**
See individual commits for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1115)

### CODE REVIEW

File: reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt [hunk 1]
```diff
@@ -527,14 +540,22 @@ class StaticHtmlReporter : Reporter() {
...
+                                    } else {
+                                        "<li><p>${it.description.replace("\n", "<br/>")}</p></li>"
+                                    }
                                 }}
                             </ul></td>
                             <td><ul>
                                 ${row.scanErrors.joinToString("\n") {
-                                    "<li><p>${it.description.replace("\n", "<br/>")}</p>" +
+                                    if (it.isResolved) {
+                                    "<li class=\"ort-addressed\"><p>${it.description.replace("\n", "<br/>")}</p>" +
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt]: Nit: Wrong indentation.

File: reporter/src/funTest/assets/file-counter-expected-scan-report.html [hunk 2]
```diff
@@ -293,7 +306,7 @@ <h3>Packages</h3>
         
     </ul></td>
     <td><ul>
-        <li><p>n/a: FileCounter - DownloadException: No source artifact URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'.<br/>Caused by: DownloadException: No VCS URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'. Please make sure the release POM file includes the SCM connection, see: https://docs.gradle.org/current/userguide/publishing_maven.html#example_customizing_the_pom_file</p><p></p></li>
+        <li><p>n/a: FileCounter - DownloadException: No source artifact URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'.<br/>Caused by: DownloadException: No VCS URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'. Please make sure the release POM file includes the SCM connection, see: https://docs.gradle.org/current/userguide/publishing_maven.html#example_customizing_the_pom_file</p></li>
```
Comment (sschuberth) [inline reporter/src/funTest/assets/file-counter-expected-scan-report.html]: Nit: Ideally, this unrelated fix would have been done in a separate commit.

File: reporter/src/main/kotlin/reporters/ReportTableModel.kt [hunk 3]
```diff
@@ -214,6 +214,6 @@ data class ReportTableModel(
             val source: String,
             val description: String,
             val resolutionDescription: String,
-            val isResolved: Boolean
+            val isAddressed: Boolean
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/ReportTableModel.kt]: Just FYI, personally I'd prefer to stick with "resolved", which also is the term usually used by ticket / issue systems.

File: reporter/src/funTest/assets/file-counter-expected-scan-report.html [hunk 4]
```diff
@@ -156,6 +156,19 @@
                     color: #2c662d;
                   }
 
+                  .ort-report-table.ort-packages tr.ort-error {
```
Comment (mnonnenmacher) [inline reporter/src/funTest/assets/file-counter-expected-scan-report.html]: Commit message nit: "the are" -> "they are"

File: reporter/src/main/kotlin/reporters/ReportTableModel.kt (hunk 3 above)
Comment (mnonnenmacher) [reply reporter/src/main/kotlin/reporters/ReportTableModel.kt]: Same here, it seems odd to use "resolution" in the configuration and call it "addressed" here. Also I'm not sure if "addressed" has the right meaning, from a dictionary:
* "address": think about and begin to deal with (an issue or problem)
* "resolve": settle or find a solution to (a problem or contentious matter)

I think the latter matches better what error resolutions are there for.

Comment (adambhere) [reply reporter/src/main/kotlin/reporters/ReportTableModel.kt]: This is what dictionaries say. However, if I say, "I've addressed a problem", the common understanding is that I've dealt with it and it no longer needs attention. Technically you could regard 'resolve' as being more precise, because it says that a resolution to the problem has been found and applied, which of course does not cover problems that turn out not to require a solution, but are listed as problems until someone has 'addressed' them. If you want to avoid changing some existing sense or usage, I wouldn't argue.
